src/main/scala/za/co/absa/hyperdrive/trigger/persistance/SensorRepository.scala

```diff
@@ -50,7 +50,11 @@ class SensorRepositoryImpl extends SensorRepository {
     }).result
   }
 
-  override def getChangedSensors(originalSensors: Seq[Sensor])(implicit ec: ExecutionContext): Future[Seq[Sensor]] = db.run {(
+  override def getChangedSensors(originalSensors: Seq[Sensor])(implicit ec: ExecutionContext): Future[Seq[Sensor]] = {
+    Future.sequence(originalSensors.grouped(100).toSeq.map(group => getChangedSensorsInternal(group))).map(_.flatten)
```

Comment:
`100` is a magic constant. I'd move it to `Configs.scala`

Reply:
Ok

Reply:
Done